defaults/main.yml

```diff
@@ -26,3 +26,4 @@ autorestic_info: false
 autorestic_check: false
 autorestic_cron: false
 autorestic_state: present
+autorestic_verbose: false
```

Comment:
Seeing as this is specifically related to the autorestic cron job, would we be able to make this variable name `autorestic_cron_verbose` ?

Also, then it pairs nicely with the `autorestic_cron` variable and we can have one after the other in the file. 🙂

Reply:
@dbrennand I was actually already thinking I should have used this naming scheme a few minutes after I submitted the PR but second guessed myself. I will make these changes shortly!